### ISSUE TITLE
Paste/drag-and-drop image polishing

### ISSUE THREAD

**Issue by juliusknorr** [nextcloud/text#2275]
For some smaller polishing it would be very nice to have some immediate feedback when pasting or dragging an image into a document. Currently there is only a loading indicator after the file was uploaded. Getting a placeholder into the document with the loading indicator might be trickier but would improve the user experience quite a lot.

cc @eneiluj @max-nextcloud @vinicius73

**Comment by max-nextcloud** [comment]
I agree. We played with the background color of the editor a bit as that's what the files app does when files are dragged in.
It did look quite nice. Not sure if it did not make it in. @eneiluj - I think you were experimenting with this, right?

**Comment by julien-nc** [comment]
Sorry for the very late answer.

There already is an immediate indicator: the menubar icon changes to a loading spinner while uploading an attachment.

I like the idea of a placeholder. Can we display something in the document without writing stuff in its content?
Or maybe we can insert something that looks like the loading ImageView and which is deleted when the upload ends.

@max-nextcloud The editor background color changes when dragging something over. This is merged. So it looks nice/ok when dragging something over but the background color goes back to normal when the upload starts. So there is no "upload in progress" indicator in the editor during the upload :grin:. Just before and after.

@juliushaertl Should we schedule that for 26?
